src/components/FiatValue.vue

```diff
@@ -16,15 +16,19 @@ export default {
   props: {
     amount: { type: [String, Number], default: 0 },
   },
-  computed: {
-    ...mapState(['settings', 'currencyRates']),
-    fiatValue() {
-      const rate = this.currencyRates.aeternity[this.settings.currency];
-      if (!this.amount) {
-        return 0;
-      }
-      return (this.amount * rate).toLocaleString('en-US', { style: 'currency', currency: this.settings.currency });
+  computed: mapState({
```

Comment:
Should we continue using this syntax?

Reply:
This is a vuex feature. They wouldn't implement that if it has an obviously bad readability or so.